cmd/bucket-handlers.go

```diff
@@ -278,15 +278,6 @@ func (api ObjectAPIHandlers) DeleteMultipleObjectsHandler(w http.ResponseWriter,
 		return
 	}
 
-	logger.Info(fmt.Sprintf("delete objects len: %d, new name: %#+v", len(deleteObjects.Objects), deleteObjects.Objects))
-
-	// Convert object name delete objects if it has `/` in the beginning.
-	for i := range deleteObjects.Objects {
-		newName := path.Join(prefix, trimLeadingSlash(deleteObjects.Objects[i].ObjectName))
-		logger.Info(fmt.Sprintf("old name: %s, new name: %s", deleteObjects.Objects[i].ObjectName, newName))
```

Comment:
It would be better to use Debug method or at least to make this a bit more informative (it will be hard to understand what does these logs mean when looking just at service logs)

Reply:
this log is deleted